stdlib/3/xmlrpc/client.pyi

```diff
@@ -0,0 +1,269 @@
+    def timetuple(self) -> time.struct_time: ...  # undocumented
+    def decode(self, data: Any) -> None: ...
+    def encode(self, out: IO[str]) -> None: ...
+
+def _datetime(data: Any) -> DateTime: ...  # undocumented
+def _datetime_type(data: str) -> datetime: ...  # undocumented
+
+class Binary:
+
+    data: bytes
+
+    def __init__(self, data: Optional[bytes] = ...) -> None: ...
+    def decode(self, data: bytes) -> None: ...
+    def encode(self, out: IO[str]) -> None: ...
+
+def _binary(data: bytes) -> Binary: ...  # undocumented
+
+WRAPPERS: Tuple[Type[DateTime], Type[Binary]]  # undocumented
+
+class ExpatParser:  # undocumented
+
+    def __init__(self, target: Unmarshaller) -> None: ...
+    def feed(self, data: Union[Text, bytes]) -> None: ...
+    def close(self) -> None: ...
+
+class Marshaller:
+
+    dispatch: Dict[Type[Any], Callable[[Marshaller, Any, IO[str]], None]] = ...  # TODO: Replace 'Any' with some kind of binding
+
+    memo: Dict[Any, None]
+    data: None
+    encoding: Optional[str]
+    allow_none: bool
+
+    def __init__(self, encoding: Optional[str] = ..., allow_none: bool = ...) -> None: ...
+    def dumps(self, values: Union[Fault, Iterable[_Marshallable]]) -> str: ...
+    def __dump(self, value: Union[_Marshallable]) -> None: ...  # undocumented
+    def dump_nil(self, value: None, write: IO[str]) -> None: ...
+    def dump_bool(self, value: bool, write: IO[str]) -> None: ...
+    def dump_long(self, value: int, write: IO[str]) -> None: ...
+    def dump_int(self, value: int, write: IO[str]) -> None: ...
+    def dump_double(self, value: float, write: IO[str]) -> None: ...
+    def dump_unicode(self, value: str, write: IO[str], escape: Callable[[str], str] = ...) -> None: ...
+    def dump_bytes(self, value: bytes, write: IO[str]) -> None: ...
+    def dump_array(self, value: Union[Iterable[_Marshallable]], write: IO[str]) -> None: ...
+    def dump_struct(self, value: Dict[str, _Marshallable], write: IO[str], escape: Callable[[str], str] = ...) -> None: ...
+    def dump_datetime(self, value: _XMLDate, write: IO[str]) -> None: ...
+    def dump_instance(self, value: Union[DateTime, Binary], write: IO[str]) -> None: ...
+
+class Unmarshaller:
+
+    dispatch: Dict[str, Callable[[Unmarshaller, str], None]] = ...
+
+    _type: Optional[str]
+    _stack: List[_Marshallable]
+    _marks: List[int]
+    _data: List[str]
+    _value: bool
+    _methodname: Optional[str]
+    _encoding: str
+    append: Callable[[Any], None]
+    _use_datetime: bool
+    _use_builtin_types: bool
+
+    def __init__(self, use_datetime: bool = ..., use_builtin_types: bool = ...) -> None: ...
+    def close(self) -> Tuple[_Marshallable, ...]: ...
+    def getmethodname(self) -> Optional[str]: ...
+    def xml(self, encoding: str, standalone: Any) -> None: ...  # Standalone is ignored
+    def start(self, tag: str, attrs: Dict[str, str]) -> None: ...
```

Comment:
It is, but I typed it to match where it's set, as a StartElementHandler in pyexpat. In theory, as Unmarshaller is public, I figured a subclass / extension might prefer the type it will actually get